satellite/audit/observer_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"storj.io/storj/pkg/storj"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+
+	"storj.io/storj/internal/memory"
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/satellite/audit"
+	"storj.io/storj/storage"
+)
+
+// For every node in testplanet:
```

Comment:
The setup steps are missed in this comment block (upload 5 files, create new audit observer, etc...)